Sources/TSCUtility/IndexStore.swift

```diff
@@ -14,9 +14,24 @@ import TSCBasic
 public final class IndexStore {
 
     public struct TestCaseClass {
+        public enum TestMethod: Hashable, Comparable {
+            case standard(String)
+            case async(String)
+
+            public var name: String {
+                switch self {
+                case .standard(let name):
+                    return name
+                case .async(let name):
+                    return name
+                }
+            }
+        }
+
         public var name: String
         public var module: String
-        public var methods: [String]
+        public var testMethods: [TestMethod]
+        @available(*, deprecated, message: "use testMethods instead") public var methods: [String]
```

Comment:
I'm not too sure what the source compatibility requirement on this is. Let me know if this should be different!